### PR TITLE
Cleaned up codebase and reverted max character count per line back to 80 characters per line.

### PR DESCRIPTION
**Pull request by armcburney**
@davepagurek, @icechen1 

It's a big diff, so I apologize for that - but I just wanted to clean up the codebase a bit since there was a lot of stuff subject to change (like language spec, etc.).

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Metrics/LineLength:
-  Max: 120
+  Max: 80
```
Comment (davepagurek) [inline .rubocop.yml]: I'm personally a fan of having slightly longer lines, we don't all have 800x600 screens any more!

File: src/ruby/Main.rb [hunk 2]
```diff
@@ -17,13 +17,12 @@
...
-    class_option :language # specify what templating engine you want to transpile to
+    class_option :beautify, aliases: 'b'
 
-    # main emerald option, processes the emerald file, generates an abstract
+    # Main emerald option, processes the emerald file, generates an abstract
     # syntax tree based on the output from the preprocessing.
-    desc 'process', 'Process a file or folder (recursively) and convert it to emerald.'
-    # option :name - TODO remember how to do method specific options.
+    desc 'process', 'Process a file or folder (recursively) and converts it to'\
+                    'emerald.'
```
Comment (davepagurek) [inline src/ruby/Main.rb]: e.g. for lines like this (in reference to the line length thing)

File: src/ruby/nodes/TagStatement.rb [hunk 3]
```diff
@@ -7,7 +7,8 @@
 # A tag
 class TagStatement < Node
   def to_html(_context)
-    "<#{elements[0].text_value}>#{elements[2].text_value}</#{elements[0].text_value}>"
+    "<#{elements[0].text_value}>#{elements[2].text_value}"\
+    "</#{elements[0].text_value}>"
```
Comment (davepagurek) [inline src/ruby/nodes/TagStatement.rb]: Maybe separate with + instead?
```
"<#{elements[0].text_value}>" +
  elements[2].text_value +
  "</#{elements[0].text_value}>"
```

File: .rubocop.yml (hunk 1 above)
Comment (armcburney) [reply .rubocop.yml]: Lol, oh you.

Comment (armcburney) [reply .rubocop.yml]: Alright, I'll change it for you bbgurl. 😍

**Review by davepagurek — APPROVED** [review]
👌

**Pull request merged**